public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -246,7 +246,13 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel,
       new behaviors.LiveNowTimer({ enabled: oldModel.liveNow }),
       preserveDashboardSceneStateInLocalStorage,
       new ScopesFacade({
-        handler: (facade) => sceneGraph.getTimeRange(facade).onRefresh(),
+        handler: (facade) => {
+          if (oldModel.meta.reloadOnScopesChange && oldModel.uid) {
+            locationService.reload();
+          } else {
+            sceneGraph.getTimeRange(facade).onRefresh();
+          }
+        },
```

Comment:
The idea of the `ScopesFacade` is to be decoupled from the dashboards and be used by others areas that are Scenes-based (i.e. Explore if they migrate to Scenes, apps etc.).

Reply:
ah, maybe it can emit an event that is subscribed to by DashboardScene? or we can create DashboardsScopesFacade that extend it? 

would be nice to keep state logic inside the state objects rather than where they are constructed (should make it easier to find logic & relevant tests). 

Or just start with these two behaviors as state options: 

```ts
  new ScopesFacade({ refreshTimeOnChange: true, reloadRouteOnChange: meta.reloadOnScopesChange });
```

Reply:
Great ideas! I'll explore them today and pick one of them. I think extending the facade is what I like the most 🤔

Reply:
Just went with extending the facade.